solidity/test/bridge/Bridge.Parameters.test.ts

```diff
@@ -1769,4 +1771,65 @@ describe("Bridge - Parameters", () => {
       })
     })
   })
+
+  describe("updateTreasury", () => {
+    const newTreasury = "0x4EDCe37af1c6e2CDfe086D7940C5a8AAde9c72e3"
+
+    context("when caller is the contract guvnor", () => {
+      before(async () => {
+        // TODO: We transfer the ownership of the Bridge governance from the
```

Comment:
But wouldn't updating through the BridgeGovernance be better aligned with a real-world scenario where the owner of the Bridge is BridgeGovernance https://github.com/keep-network/tbtc-v2/blob/main/solidity/deploy/17_transfer_bridge_governance.ts#L10 ?

Reply:
This is a real-world scenario only until the `BridgeGovernance` is the owner of `Bridge` :) There is no guarantee this will always be the case. So I think we should test `Bridge` directly in unit tests for `Bridge` and test `BridgeGovernance + Bridge` combo in unit tests for `BridgeGovernance`. This way we can be sure that the validation is in the `Bridge.updateTreasury(...)` and not in `BridgeGovernance.finalizeTreasuryUpdate()`.